src/BasePlatform.ts

```diff
@@ -258,7 +258,9 @@ export default abstract class BasePlatform {
         return null;
     }
 
-    setLanguage(preferredLangs: string[]) {}
+    async setLanguage(language: string) {
+        throw new Error("Unimplemented");
+    }
```

Comment:
languageHandler expects this to accept a string array: https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/languageHandler.tsx#L313